trace-mini-agent/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "datadog-trace-mini-agent"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+hyper = { version = "0.14", features = ["full"] }
+tokio = { version = "1", features = ["full"] }
```

Comment:
do we need full features for hyper and tokio?

Reply:
fixed with just the features we need.